config/form-content/bestuursorgaan-extended-example/form.ttl

```diff
@@ -0,0 +1,51 @@
+@prefix form: <http://lblod.data.gift/vocabularies/forms/>.
+@prefix sh: <http://www.w3.org/ns/shacl#>.
+@prefix displayTypes: <http://lblod.data.gift/display-types/>.
+@prefix schema: <http://schema.org/>.
+@prefix ext: <http://mu.semte.ch/vocabularies/ext/>.
+@prefix locn: <http://www.w3.org/ns/locn#>.
+@prefix mu: <http://mu.semte.ch/vocabularies/core/>.
+
+ext:emailF
+    a form:Field;
+    form:displayType displayTypes:defaultInput;
+    ext:extendsGroup ext:bestuursorgaanPG;
+    sh:name "Email";
+    sh:order 301;
+    sh:path schema:email.
+ext:telefoonF
+    a form:Field;
+    form:displayType displayTypes:defaultInput;
+    ext:extendsGroup ext:bestuursorgaanPG;
+    sh:name "Telefoon";
+    sh:order 302;
+    sh:path schema:telephone.
+ext:adresF
+    a form:Field;
+    form:displayType displayTypes:addressSelector;
+    ext:extendsGroup ext:bestuursorgaanPG;
+    sh:name "Address";
+    sh:order 303;
+    sh:path locn:address.
+
+<http://data.lblod.info/id/lmb/forms/bestuursorgaan-extended-example>
+    a form:Extension;
+    form:includes
+        ext:emailF,
+        ext:telefoonF,
+        ext:adresF;
+    ext:extendsForm <http://data.lblod.info/id/lmb/forms/bestuursorgaan>;
+    ext:extendsGroup ext:bestuursorgaanPG;
```

Comment:
for this one i think we should still use sh:group. will clarify in the readme of the form builder

Reply:
in fact! I think you don't even need this as this property group is already part of the existing form, so you'll have this link specified already that way.